### DISCUSSION TITLE
Latest DNN and Latest Community Forums Extension in-compatible?

### DISCUSSION THREAD

**Discussion by KylePerenchio** [DNNCommunity/Dnn.CommunityForums#1183] [Q&A]
Community Forums version 8.01.02 seems not to be compatible with DNN 9.13.06.  I installed it fresh with a fresh DNN install and no matter what I do or edit I cannot remove the default forms created with installation. In-Fact I cant delete anything for that matter as administrator or any other user.  Am I missing something? Is this a known issue?

**Comment by johnhenley** [comment]
Hi @BennuFury. This is an issue (#1156) in Forums 08.01.02, and is fixed (#1157) in the next release 08.02.00, which should be released soon. For now, you can set the forums as 'hidden' which will keep them from showing for non-administrators. You can also achieve that by removing security access for all security roles.
